web/src/util/formats.js

```diff
@@ -1,9 +1,10 @@
 import { format } from 'd3-format';
 
-const isNumeric = n => !Number.isNaN(parseFloat(n)) && Number.isFinite(n);
-
 const BIG_NUMBER_THRESHOLD = 1000000;
 
+// eslint-disable-next-line no-restricted-globals
+export const isNumeric = n => !isNaN(parseFloat(n)) && isFinite(n);
```

Comment:
Why the change from `Number.isNaN` to the older version?

Reply:
our old fn was saying that `'0.00'` was not numeric -- this one solves that, though now I see that it's due to the `isFinite` change...

![image](https://user-images.githubusercontent.com/1060893/44662238-2ea07600-a9db-11e8-8504-be41f7e0fcf0.png)

Reply:
What if we simplified to:

```js
export const isNumeric = n => Number.isFinite(Number.parseFloat(n));
```

It turns out that `NaN` is not considered finite, the `isFinite` check should be enough, and then we get to use the scoped methods instead of the global ones.  Since this PR is already merged, we can put this on the backburner if it makes sense and change it sometime in the future when we happen to be fiddling in the utils file again.